Script/AtomicEditor/ui/frames/inspector/NodeInspector.ts

```diff
@@ -269,6 +269,8 @@ class NodeInspector extends ScriptWidget {
 
                     prefabComponent.breakPrefab();
 
+                    this.sendEvent("EditorActiveNodeChange", { node: this.node });
```

Comment:
Break prefab also remove a PrefabComponent. I think it's ok to leave that for now, until we will refactor editor inspectors. The same thing for save/undo buttons on prefab UI layout, they send EditorActiveNodeChange event while they shouldn't do it